### ISSUE TITLE
automatic download not working on synology

### ISSUE THREAD

**Issue by totalcarnage** [SiCKRAGETV/sickrage-issues#3188]
SR Version: BRANCH: (master) / COMMIT: (638270689679f3958d1b593c329e7c6e5cb7448e) 
Python Version: 2.7.10 (default, Jun 3 2015, 20:43:13) [GCC 4.7.3 20130102 (prerelease)]
OS: Synology DS1815+ (DSM 5.2-5592 Update 4)

1) SR wont download new "wanted" files automaticaly. this used to work fine a few days ago.

2) some files are being downloaded if i manualy klick the search button in SR except:

2.1) if a show was shown on the i.e. 06.10.2015 and i try to download them on the same day manualy with SR it wont work - even if my set up search provider has them. (i checked on the site)
if i try the same show a day later - it works.

my timezone: CEST (Central European Summer Time) UTC/GMT +2 hours

My Search Settings:
download propers - check
check propers every - 24h
backlog search frequency - 720
daily search fequency - 30
allow high priority - check

search privders:
tpb
kat

i already show down SR and deleted cache.db and even removed the content of the cache folder except the images dir - withour any success.

maybe the problem has something to do with the error i get after i restart SR which is already know to you guys
__________________________error after restart_______________________________
2015-10-07 14:00:39 Thread-13 :: Failed doing webui callback: Traceback (most recent call last):
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/sickbeard/webserve.py", line 265, in async_call
    result = function(**kwargs)
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/sickbeard/webserve.py", line 660, in index
    t = PageTemplate(rh=self, file="home.mako")
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/sickbeard/webserve.py", line 116, in __init__
    super(PageTemplate, self).__init__(*args, **kwargs)
  File "/usr/local/sickbeard-custom/var/SickBeard/lib/mako/template.py", line 321, in __init__
    module = self._compile_from_file(path, filename)
  File "/usr/local/sickbeard-custom/var/SickBeard/lib/mako/template.py", line 379, in _compile_from_file
    module = compat.load_module(self.module_id, path)
  File "/usr/local/sickbeard-custom/var/SickBeard/lib/mako/compat.py", line 66, in load_module
    return imp.load_source(module_id, path, fp)
ImportError: Loaded module _usr_local_sickbeard_custom_var_SickBeard_gui_slick_views_home_mako not found in sys.modules

**Comment by SiCKRAGEB0T** [comment]
## Thanks for the bug/issue report
Please make sure to submit all of the [required information](https://github.com/SiCKRAGETV/sickrage-issues/blob/master/README.md), or we won't be able to help you!
Provide as **much** information as you can such as debug logs, commit hash, and tell us how to replicate the problem.For basic **questions** use the [ [forum] ](https://sickrage.tv/forums/), and for **Feature Requests** [ [Feathub] ](http://feathub.com/SiCKRAGETV/SickRage)
Also make sure the title accurately describes your issue. ("SR not working" etc. is not ok!)

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist)

## Many issues can simply be solved by:
- Making sure you update to the latest version.
- Have you tried turning your device off and on again?
- Using the **search** function to see if this issue has already been reported/solved.
- Do a [GIT reset](https://github.com/SiCKRAGETV/sickrage-issues/wiki/FAQ%27s-and-Fixes#update-problems-try-this).
- Checking the [ [Wiki] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki) for [ [Guides] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Installation-&-Configuration-Guides), [ [FAQS] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/FAQ%27s-and-Fixes) [ [Main settings] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Settings-explained), [ [Show settings] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Show-settings-explained) [ [Remaining settings] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Remaining-settings-explained)
- Shutting down SickRage and removing the `cache/mako` & `cache/sessions` folders.
- If SickRage doesnt find any episodes you probably need to :
 - add a [Scene Exception](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Scene-exceptions-and-numbering)
 - Set the [Default Episode Status](https://github.com/SiCKRAGETV/sickrage-issues/wiki/FAQ%27s-and-Fixes#newly-aired-shows-are-not-downloading-and-set-to-skippedignored) to `WANTED`
 - Make sure you understand how [quality detection](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Quality-Settings#quality-detectiondetermination) works during searches.
 - Shutdown SickRage and remove the `cache.db` file.
- If you use a QNAP and have SSL errors try this [package](http://apps.qnap.community/11-community/17-qsickrage).
- If you have a `'ascii' codec can't encode character` error, than set your Locale to [UTF-8](https://github.com/SiCKRAGETV/sickrage-issues/wiki/FAQ%27s-and-Fixes#i-have-problems-with-special-characters-%C3%A9-etc-what-can-i-do)

*ps. If you enjoy Sickrage and like the support provided, than please consider making a [donation](https://github.com/SiCKRAGETV/SickRage/wiki/Donations) to keep SickRage alive.*

**Comment by miigotu** [comment]
The only problem identified so far with providers not working is when using a proxy. Are you using a proxy? I suggest using rarbg until we figure it out.

**Comment by totalcarnage** [comment]
unfortunately it doenst work with rargb either.
the fun thing with tpb and kat is that im pretty sure it'll let me download manualy via SR tomorrow - like it did today with the episodes from yesterday.

**Comment by fernandog** [comment]
Loaded module _usr_local_sickbeard_custom_var_SickBeard_gui_slick_views_home_mako not found in sys.modules

**Comment by miigotu** [comment]
@totalcarnage likely those torrents had 0 seeds the first time around and were ignored?

**Comment by totalcarnage** [comment]
thx for the effort @miigotu but there are more than 100 seeds the first time around.
thought about that too and went directly to the site to check with a completeanual download without SR and it worked.

nope, not using any proxy's

**Comment by SanctiMortem** [comment]
Have the same problem here, also made sure my provides had them and it did with almost 7k seeders and it also works manually after a day or two. I've had this since I did the las update a couple of days ago. And I don't use any proxys.
I had a issue open but I guess there are too many with the same probkem that it got closed. I've tried everything that I've read here but no luck. Just updated again 10 minutes ago but the problem persists. Hope we can find a solution soon!! 
Thanks for the help!
